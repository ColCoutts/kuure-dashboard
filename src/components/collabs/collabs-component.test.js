import React from 'react';
import { shallow } from 'enzyme';
import Collabs from './collabs-component';

describe('collabs component', () => {
  it('renders a snapshot', () => {
    const wrapper = shallow(<Collabs />);
    expect(wrapper).toMatchSnapshot();
  });
});
