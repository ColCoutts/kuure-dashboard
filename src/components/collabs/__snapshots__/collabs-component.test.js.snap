// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`collabs component renders a snapshot 1`] = `ShallowWrapper {}`;
